packages/twenty-docs/docs/user-guide/basics/opportunities.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Opportunities
+sidebar_position: 1
+sidebar_custom_props:
+  icon: TbTargetArrow
+---
+
+All opportunities are presented in a Kanban board, where each column represents the stage of your workflow and each card represents a record. For each card, you can list the amount, close date, probability, and the point of contact. You can also move each card between stages as it goes through your workflow. 
+
+<div style={{textAlign: 'center'}}>
+    <img src="/img/user-guide/all-opportunities.png" alt="All opportunities"/>
```

Comment:
The image where previously added in light mode (for custom objects) ; now they're in dark mode. Could you please add bot and adapt with the theme? Or at least if you don't have time to add both keep it consistent and add light-mode screenshots only